src/helpers/getFormattedDate.ts

```diff
@@ -0,0 +1,2 @@
+export const getFormattedDate = (date: string = new Date().toString()) =>
+  new Date(date).toISOString().split("T")[0];
```

Comment:
personally I don't like this splitting ISO dates, fe example we have new Date().toLocaleDateString()